front/src/applications/editor/tools/rangeEdition/tool-factory.tsx

```diff
@@ -204,7 +204,25 @@ function getRangeEditionTool<T extends EditorRange>({
       const feature = (e.features || [])[0];
 
       if (isOnModeMove(interactionState.type)) {
-        setState({ interactionState: { type: 'idle' } });
+        if (interactionState.type === 'moveRangeExtremity' && entity.properties.track_ranges) {
+          // after resizing a track range, check if the user dragged an extremity beyond the other one
+          // if he did, switch the values of each extremities
+          const updatedTrackIndex = interactionState.rangeIndex;
+          const updatedTrack = entity.properties.track_ranges[updatedTrackIndex];
+
+          if (updatedTrack.begin > updatedTrack.end) {
+            const currentBegin = updatedTrack.begin;
+            updatedTrack.begin = updatedTrack.end;
+            updatedTrack.end = currentBegin;
```

Comment:
[updatedTrack.begin, updatedTrack.end] = [updatedTrack.end, updatedTrack.begin]

Reply:
Oh didn't think about that, nice !